core/components/organisms/textField/TextFieldWithTextarea.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Textarea, Label } from '@/index';
+import { BaseProps } from '@/utils/types';
+import { TextareaProps } from '@/index.type';
+import { renderHelpText, renderCounter } from './TextFieldCommon';
+
+export interface TextFieldWithTextareaProps extends BaseProps {
+  /**
+   * Label of the `Input`
+   */
+  label?: string;
+  /**
+   * Set as `true` to show TextField with Textarea
+   */
+  withTextarea?: boolean;
+  /**
+   * it defines the maximum number of characters
+   * (as UTF-16 code units) the user should ideally enter into the Textarea.
+   * @default 200
+   */
+  max?: number;
+  /**
+   * Add text below `input`
+   */
+  helpText?: string;
+}
+
+export type TextFieldTextareaProps = TextFieldWithTextareaProps & TextareaProps;
+
+export const TextFieldWithTextarea = (props: TextFieldTextareaProps) => {
+  const { label, rows = 3, resize = true, required, error, onChange, value = '', max = 200, helpText = ' ' } = props;
+
+  const textareaRef = React.useRef(null);
+  const [inputText, setInputText] = React.useState<string>(value);
+  const [helptextWidth, setHelptextWidth] = React.useState(0);
+
+  const onChangeHandler = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(e.target.value);
+    if (onChange) onChange(e);
+  };
+
+  const inputError = error || inputText.length > max;
+
+  React.useEffect(() => {
+    const textarea = textareaRef.current;
+    const resizeObserver = new (window as any).ResizeObserver((entries: any) => {
```

Comment:
Please use some event of textarea or add polyfill for IE 11 support.

Reply:
Alternatively, check if resize observer is available then only attach the resizing capability.